.github/workflows/compatibility.yml

```
@@ -4,7 +4,7 @@ on:
   pull_request
 
 env:
-  WC_L2_VERSION: '7.0.1'
+  WC_L2_VERSION: '7.1.0'
```

The newest WC `7.1` version is `7.1.1`

```suggestion
  WC_L2_VERSION: '7.1.1'
```